config/master.d/master.conf

```diff
@@ -4,6 +4,7 @@ interface: 0.0.0.0
 event_return: mysql
 presence_events: True
 worker_threads: 10
+timeout: 20
```

Comment:
this is not a job timeout - it is a time out for the salt master to connect to minions, and minions connecting to the salt master. see conversation on #126